presto-docs/src/main/sphinx/security/salesforce.rst

```diff
@@ -0,0 +1,80 @@
+============================
+Salesforce Authentication
+============================
+
+Presto can be configured to enable frontend password authentication over
+HTTPS for clients, such as the CLI, or the JDBC and ODBC drivers. The
+username and password (or password and :ref:`security tokens` concatenation)
```

Comment:
See explanation below
```
username and password (or password and `security tokens <#security-tokens>`__ concatenation)

Reply:
Cool.  Honestly, I wanted to do this, but couldn't quite figure out how.  Thanks.